.github/workflows/staticbuild.yml

```diff
@@ -0,0 +1,46 @@
+name: staticbuilds
+
+on:
+    # release:
+    #     types: [published]
+    # schedule:
+    # - cron: "14 13 * * *"
+    workflow_dispatch:
+
+env:
+  CTEST_PARALLEL_LEVEL: "1"
+  CMAKE_BUILD_PARALLEL_LEVEL: "4"
+
+jobs:
+  staticbuilds:
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+
+    steps:
+    - uses: dtolnay/rust-toolchain@1.70
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: Prepare
+      run: |
+           rustup target add x86_64-unknown-linux-musl
+           rustup target add aarch64-unknown-linux-musl
+           sudo apt install musl-tools crossbuild-essential-arm64 -y
+    - name: Build
+      run: |
+           CMAKE_WITH_GETTEXT=0 CC=aarch64-linux-gnu-gcc RUSTFLAGS="-C linker=aarch64-linux-gnu-gcc -C link-arg=-lgcc" cargo build -r --target aarch64-unknown-linux-musl
+           cargo build -r --target x86_64-unknown-linux-musl
```

Comment:
better to use long options like `--release` instead of `-r` here for readability